File: 2015 ICPC North American Qualifier Contest/simonsays.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int times = scanner.nextInt();
        scanner.nextLine(); // consume newline
        
        for(int i = 0; i < times; i++) {
            String s = scanner.nextLine();
            
            if(s.startsWith("Simon says")) {
                String result = s.substring(11);
                System.out.println(result);
            }
        }
        
        scanner.close();
    }
}

File: 2015 ICPC North American Qualifier Contest/simonsays.cs
using System;

class Program {
    static void Main() {
        int times = int.Parse(Console.ReadLine());
        
        for(int i = 0; i < times; i++) {
            string s = Console.ReadLine();
            
            if(s.StartsWith("Simon says")) {
                string result = s.Substring(11);
                Console.WriteLine(result);
            }
        }
    }
}